.github/workflows/intel.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 # intel-basekit intel-hpckit are too large in size
   build_icc:
     name: oneAPI ICC SP&DP
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

Comment:
Please remove the "old" ICC, it is EOL since mid 2023 and I already removed it from pyAMReX.
https://github.com/AMReX-Codes/pyamrex/pull/409

Intel only supports ICX (oneAPI compilers) going forward.

Reply:
Done in [43e0a32](https://github.com/ECP-WarpX/WarpX/pull/5731/commits/43e0a32839273c88e961ebdf264ddda45d8c6ab5).

Reply:
Now that the `oneAPI ICC SP&DP` workflow job has been removed, it should aslo be removed from the list of "required" workflows in the repository settings by one of the owners.